setup.py

```diff
@@ -44,8 +44,8 @@ def find_version(*file_paths):
     name='awscli',
     version=find_version("awscli", "__init__.py"),
     description='Universal Command Line Environment for AWS.',
-    long_description=read('README.md'),
-    long_description_content_type="text/markdown",
+    long_description=read('README.rst'),
+    long_description_content_type="text/x-rst",
```

Comment:
Let's just remove this line to revert back completely to what the `setup.py` used to have: https://github.com/aws/aws-cli/commit/08ade7344b862ab3974f442bdd5580d81086c7a7